packages/ckeditor5-fullscreen/tests/manual/fullscreen.ts

```diff
@@ -52,6 +52,7 @@ import CloudServices from '@ckeditor/ckeditor5-cloud-services/src/cloudservices.
 import Style from '@ckeditor/ckeditor5-style/src/style.js';
 import GeneralHtmlSupport from '@ckeditor/ckeditor5-html-support/src/generalhtmlsupport.js';
 import Bookmark from '@ckeditor/ckeditor5-bookmark/src/bookmark.js';
+import PresenceList from '@ckeditor/ckeditor5-real-time-collaboration/src/presencelist.js';
```

Comment:
This manual test throws for me right now as there is no `.presence` element in HTML, but let's only use open-source features in the manual test (I'm not sure if it even worked on CI) and premium features - using `all-features-rtc` manual test from commercial repo.